iDiff/utils/image_prep_utils.go

```diff
@@ -89,8 +89,11 @@ type CloudPrepper struct {
 }
 
 func (p CloudPrepper) ImageToFS() (string, error) {
+	// The regexp when passed a string creates a list of the form
+	// [entire string, everything after the first "/", everything after the ":" after the "/"]
```

Comment:
not the first "/" but last one actually because it's the entire URL except for the last part which is the image name (and potential tag)... but actually I don't even know if it's technically the last "/" because I think it's possible to name images with "/"? not positive about that though you might want to just rephrase that in general to be more accurate